<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClassClass</class>
 <widget class="QMainWindow" name="MyClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QImageLabel" name="pictureLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>701</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>721</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QImageLabel" name="regionLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>100</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>640</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Writing to:</string>
    </property>
   </widget>
   <widget class="QLabel" name="writeToLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>640</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>666</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image folder:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagePathLabel">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>667</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>713</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Click = +1  (lane), Right Click = -1 (not lane)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>690</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vectors trained:</string>
    </property>
   </widget>
   <widget class="QLabel" name="trainedCountLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>690</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="svmPredictCheckBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>600</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SVM Predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="svmPredictFileLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>620</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>650</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Predict Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="svmResultLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>650</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="svmPredictOverlayCheckBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>600</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlay</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_file"/>
    <addaction name="actionNew_file"/>
    <addaction name="actionSelect_image_folder"/>
   </widget>
   <widget class="QMenu" name="menuSVM_Train">
    <property name="title">
     <string>SVM Train</string>
    </property>
    <addaction name="actionChoose_Data_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSVM_Train"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
  <action name="actionNew_file">
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="actionSelect_image_folder">
   <property name="text">
    <string>Select image folder</string>
   </property>
  </action>
  <action name="actionChoose_Data_File">
   <property name="text">
    <string>Choose Data File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QImageLabel</class>
   <extends>QLabel</extends>
   <header>qimagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections/>
</ui>
